docs/Top50ServiceChecker.ipynb

```diff
@@ -69,155 +76,249 @@
+    "    \"S5P_L2__CH4___\": \"C1442068505-GES_DISC\",\n",
+    "    \"S5P_L2__SO2___\": \"C1442068508-GES_DISC\",\n",
+    "    \"MOD04_L2\": \"C1443533440-LAADS\",\n",
+    "    \"MYD04_L2\": \"C1443533683-LAADS\",\n",
+    "    \"VNP46A2\": \"C1898025206-LAADS\",\n",
+    "    \"MYD03\": \"C1379841358-LAADS\",\n",
+    "    \"MYD021KM\": \"C1379758607-LAADS\",\n",
+    "    \"MYD06_L2\": \"C1443536017-LAADS\",\n",
+    "    \"MYD35_L2\": \"C1443715587-LAADS\",\n",
+    "    \"MOD03\": \"C1379767668-LAADS\",\n",
+    "    \"MOD021KM\": \"C1378227407-LAADS\",\n",
+    "    \"MOD06_L2\": \"C1443535037-LAADS\",\n",
+    "    \"VNP46A1\": \"C1897815356-LAADS\",\n",
+    "    \"MOD07_L2\": \"C1443541366-LAADS\",\n",
+    "    \"MOD05_L2\": \"C1443531026-LAADS\",\n",
+    "    \"MOD02HKM\": \"C1378577630-LAADS\",\n",
+    "    \"MCD06COSP_D3_MODIS\": \"C1887589686-LAADS\",\n",
+    "    \"MYD04_3K\": \"C1443528505-LAADS\",\n",
+    "    \"MYD07_L2\": \"C1443544285-LAADS\",\n",
+    "    \"MOD04_3K\": \"C1443420430-LAADS\",\n",
+    "    \"MOD35_L2\": \"C1443561895-LAADS\",\n",
+    "    \"MYD05_L2\": \"C1443531357-LAADS\",\n",
+    "    \"MOD11A1\": \"C1748058432-LPCLOUD\",\n",
+    "    \"MYD11A1\": \"C1748046084-LPCLOUD\",\n",
+    "    \"MOD11A2\": \"C2269056084-LPCLOUD\",\n",
+    "    \"MYD11A2\": \"C2269057787-LPCLOUD\",\n",
+    "    \"MCD19A2\": \"C2324689816-LPCLOUD\",\n",
+    "    \"MYD09GA\": \"C2202498116-LPCLOUD\",\n",
+    "    \"MOD15A2H\": \"C2218777082-LPCLOUD\",\n",
+    "    \"MOD09GA\": \"C2202497474-LPCLOUD\",\n",
+    "    \"MCD43A4\": \"C2218719731-LPCLOUD\",\n",
+    "    \"MOD14\": \"C2271754179-LPCLOUD\",\n",
+    "    \"MCD15A2H\": \"C2222147000-LPCLOUD\",\n",
+    "    \"MYD14\": \"C2278858993-LPCLOUD\",\n",
+    "    \"MCD43A3\": \"C2278860820-LPCLOUD\",\n",
+    "    \"MOD13Q1\": \"C1748066515-LPCLOUD\",\n",
+    "    \"MYD13Q1\": \"C2307290656-LPCLOUD\",\n",
+    "    \"MOD13A3\": \"C2327962326-LPCLOUD\",\n",
+    "    \"MYD13A3\": \"C2327957988-LPCLOUD\",\n",
+    "    \"MYD09GQ\": \"C2343109950-LPCLOUD\",\n",
+    "    \"MCD15A3H\": \"C2343110937-LPCLOUD\",\n",
+    "    \"ASTGTM\": \"C1711961296-LPCLOUD\",\n",
+    "    \"ASTGTM_NC\": \"C2439422590-LPCLOUD\",\n",
+    "    \"MOD09GQ\": \"C2343115666-LPCLOUD\",\n",
+    "    \"MOD09A1\": \"C2343111356-LPCLOUD\",\n",
+    "    \"MOD16A2\": \"C2343113232-LPCLOUD\",\n",
+    "    \"MOD09Q1\": \"C2343112831-LPCLOUD\",\n",
+    "    \"MYD09Q1\": \"C2343114343-LPCLOUD\",\n",
+    "    \"MYD09A1\": \"C2343113743-LPCLOUD\",\n",
+    "    \"MOD11_L2\": \"C2343115255-LPCLOUD\",\n",
+    "    \"MYD11_L2\": \"C2343114808-LPCLOUD\",\n",
+    "    \"Daymet_Daily_V4R1_2129\": \"C2532426483-ORNL_CLOUD\",\n",
+    "    \"MODIS_A-JPL-L2P-v2019.0\": \"C1940473819-POCLOUD\",\n",
+    "    \"OSCAR_L4_OC_third-deg\": \"C2036880665-POCLOUD\",\n",
+    "    \"MUR-JPL-L4-GLOB-v4.1\": \"C1996881146-POCLOUD\",\n",
+    "    \"VIIRS_NPP-STAR-L3U-v2.80\": \"C2147485059-POCLOUD\"\n",
```

Comment:
Collection concept IDs aren't as user-friendly, but I wanted to use them to avoid the GLDAS multiple issues. I believe the concept IDs now point to v2.1 for the two GLDAS collections, rather than v2.0.